src/gdpr-banner/gdpr-banner.scss

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../dialog/index';
+
+@mixin iui-gdpr-banner {
+  @include iui-dialog;
+  border-radius: 0;
+  box-shadow: none;
+  background-color: var(--iui-color-background-3);
+  border-top: 1px solid var(--iui-color-background-border);
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
I don't think we should be specifying `right` and `left` because that makes it span the entire width. It's better to have the lines start wrapping after a certain width (e.g. `75ch`).

On an ultrawide display, this will become unnecessarily wide.

Reply:
This makes it worse because it looks fine for "medium" size screens but breaks on narrow or ultrawide screens.
![](https://user-images.githubusercontent.com/9084735/171224409-75af4d57-df34-40fc-90c0-ebc835334e0f.png)
![](https://user-images.githubusercontent.com/9084735/171224845-7cecb70e-cb67-4a1d-911c-9c10b580ebbd.png)

My suggestion is to avoiding hardcoding `left` and `right` to a specific value, and set a `max-width` instead. I think you will also need to center it after that.

Reply:
I got the max-width in, still trying to figure out how to center it